### DISCUSSION TITLE
quite high data intelligence costs $30 for 87MB data in with 20 PDFs

### DISCUSSION THREAD

**Discussion by schuerstedt** [microsoft/AI-Chat-App-Hack#45] [Q&A]
I am a little shocked by the cost of the data intelligence part. 20 PDFs, 87MB data ingestion and I got a bill of $30 for that. 

Luckily it is running on a MSDN subscription, so not real money. I have a couple of GB of PDFs, but that way I could not afford this. 

Any ideas how to make this affordable?

![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/2076412/3b565092-8326-44cf-864d-a1c98678a472)

![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/2076412/5b3a3d3d-3ed2-4ba7-b472-be7ee408f964)

**Comment by pamelafox** [comment]
Have you tried the local PDF parser? (Add --localpdfparser to prepdocs.sh) It'd be good to know if that performs well enough for your documents, since that's free.
